articles/liquity/Asset Risk Assessment - bLUSD.md

```diff
@@ -0,0 +1,203 @@
+
+In an undesirable scenario where the Stability Pool has no funds to facilitate the liquidations, the debt obligation from a Trove that is deemed to be liquidated might get transferred to other LUSD debt holders. Transferring this debt obligation will reduce the health factor (collateral ratio) of the remaining Troves making them vulnerable to liquidation if the ETH price chooses to go down.
+
+![Debt trasnfer in case of insufficient funds under Stability Pool](https://lh5.googleusercontent.com/dsMYpvcTpi81p2KC3WNCcD0cWwUfAjtGPN7Uyo3KoI_vu54ELjImim8tCw7Wjf25iVU-MvIeQPxG8_vOWiXDHCkfqKPmODh0kDXAIXz6UwJ4HnQiQGhDSMxDHqPVUK5kO4zcpfHG9lwrnS5PQY9L0n5SLS8iaF7_-z8aTca7v-xEUadX0DKl0Y7UAI487w)
+
+> The redistribution of the collateral and debt is done in proportion to the recipient Trove’s collateral amount. This means that Troves which are heavily collateralized will receive more debt and collateral from liquidated positions than those with low collateralization, ensuring that the system does not create cascading liquidations.
+>
+> **Liquity Whitepaper**
+
+The above ensures that when risky debt is redistributed to the existing debt holders, Recovery Mode is triggered. Recovery Mode is activated when the protocol's Total Collateral Ratio (TCR) falls below 150%. In this undesirable mode of operation, Troves with a collateral ratio between the current TCR and 110% also become subject to liquidation.
+
+These liquidations are carried at 110% CR and excess collateral can be reclaimed by the borrower.
+
+Recovery Mode generates more liquidation with a capped discount. If the funds in the Stability Pool are not sufficient to set off these heavy liquidations, the LUSD peg might suffer.
+
+Chicken Bonds were launched to ensure there are funds available in the Stability Pool to facilitate liquidation and to ensure the LUSD peg in harsh conditions.
+
+# Chicken Bonds
+
+*[Resouces]*
+
+*[Chicken Bonds: Self-Bootstrapping Liquidity](https://docsend.com/view/dakurpcuv3259bnx)*
+
+*[Chicken Bond Docs](https://liquity.gitbook.io/chicken-bonds/faq/table-of-content)*
+
+*[A Game Theoretic Model of Chicken Bonds Dynamic](https://github.com/Risk-DAO/Reports/blob/main/Chicken%20bonds%20analysis.pdf)*
+
+Chicken Bonds are a novel mechanism for projects to bootstrap protocol-owned liquidity at no cost while boosting yield opportunities for end users. The amplified value accrual to bTKN is achieved by investing the funds under protocol-owned liquidity and the bonds’ (underlying tokens (TKN)).  
+
+Users may deposit TKN in exchange for an accruing balance of bTKN. At any time, bond-holders can either retrieve their principal foregoing the accrued amount (“chicken out”) or trade it in for the accrued bTKN (“chicken in”). Here, a portion of the TKN acquired by the system backs the bTKN supply, and this portion depends on the time of chicken_in.
```

Comment:
The piece alternates between "chicken in" "`chicken_in`" and "CI", I'd suggest making it consistent. For readability just using "chicken in" as a regular verb seems better to me.
For instance, in the last part of this sentence: "... this portion depends on the time at which the bond holder chickened in"

Reply:
CI, Chicken_in, and chicken_in were changed to chicken in